cardano-db-sync/src/Cardano/DbSync/Era/Universal/Insert/Other.hs

```diff
@@ -163,15 +164,26 @@ insertStakeAddressRefIfMissing _trce cache addr =
 
 insertMultiAsset ::
   (MonadBaseControl IO m, MonadIO m) =>
+  Trace IO Text ->
```

Comment:
Why introduce this parameter and ignore it later? ie, why not leave this out?

Reply:
it was when I was doing tracing to make sure values were correct. If only we had a monad transformer with all these common values on so we didn't have to pass them down.